Feature set override duration (#15)
lurgh/infinitive: README.md

* `infinitive/zone/X/coolSetpoint`: current cool set point, in whole degrees
* `infinitive/zone/X/heatSetpoint`: current heat set point, in whole degrees
* `infinitive/zone/X/fanMode`: current fan mode setting, Home Assistant compatible: `low`, `med`, `high`, `auto`
* `infinitive/zone/X/hold`: bool flag for permanent Hold, `false` or `true` (not really useful with HA -- use `preset` instead)

> * `infinitive/zone/X/overrideActive`: bool flag for the `hold until` override state

Reviewer: let's use quotes rather than code style for 'hold until' since it is a concept and not a literal:
```suggestion
* `infinitive/zone/X/overrideActive`: bool flag for the 'hold until' override state
```

* `infinitive/zone/X/preset`: HA-style "preset" flag; currently `hold`, `vacation`, or `none`
* `infinitive/zone/X/damperPos`: zone damper position reported by zoning unit, 0-100 as whole number percent where 100 is fully open
* `infinitive/zone/X/flowWeight`: airflow allocation factor for this zone as a decimal fraction (0-1) - multiply the total airflowCFM
  by this number to get the reported airflow for this zone.

> * `infinitive/zone/X/overrideDurationMins`: minutes remaining on zone override (`hold until`), zero if none

Reviewer: ```suggestion
* `infinitive/zone/X/overrideDurationMins`: minutes remaining on zone override ('hold until'), zero if none
```

Author: I'm not seeing a suggested change here. Is my vision playing tricks ?

Reviewer: yeah it is just changing the quote from backquote (code style) to single-quote, for consistency.  same change as suggested in the previous (unaddressed) comment.   your content and intent are fine, i am just trying the keep the style consistent

* `infinitive/zone/X/temp16`: high resolution, unsmoothed temperature, as a decimal number to .0625 degree precision

HomeAssistant MQTT Discovery topics published:
* `homeassistant/sensor/infinitive/*/config`: discovery topics, one per sensor, for:
  * all the "global" sensors: `outdoorTemp`, `humidity`, `rawMode`, `blowerRPM`, `airflowCFM`, `staticPressure`, `coolStage`, `heatStage`, `action`
  * all the vacation sensors: `vacation/active`, `vacation/days`, `vacation/hours`, `vacation/minTemp`, `vacation/maxTemp`, `vacation/minHumidity`, `vacation/maxHumidity`, `vacation/fanMode`
  * per-zone "bonus" sensors (not supported by the Climate integration): `damperPos`, `flowWeight`, `overrideDurationMins`, `temp16`
* `homeassistant/number/infinitive/*/config`: discovery topics, one per writable number entity, for:

>   * per-zone "Set override duration" in 15-minute steps, using the same MQTT state and `/set` command topics

Reviewer: Assuming we stay with using the number entity for this, reword since not enforcing steps here any more.

```suggestion
  * per-zone readable and settable numeric values: `overrideDurationMins`
```

Author: will fix

* `infinitive/zone/X/fanMode/set`: set the fan mode setting, same options as above
* `infinitive/zone/X/hold/set`: set the zone hold setting, same options as above
* `infinitive/zone/X/preset/set`: set the zone "preset" setting, `hold` or `none`; `vacation` cannot be set here but setting `hold` will unset it

> * `infinitive/zone/X/overrideDurationMins/set`: set the remaining override duration in minutes; `0` resumes schedule by clearing hold/override state

Reviewer: we don't need to mention the implementation here.

```suggestion
* `infinitive/zone/X/overrideDurationMins/set`: set the remaining override duration in minutes; `0` cancels any remaining override time and resumes schedule
```
